recipes/editabletuple/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "editabletuple" %}
```

Comment:
Every time I read edibletuple instead of edittuple. Should not review PRs on an empty stomach.